.docker/paste/bashrc.sh

```diff
@@ -0,0 +1,3 @@
+PS1="\w \$ "
+
+alias vim="vi"
```

Comment:
why is this a thing? how often do you have to attach to a container and why is that prompt format better than another?

Reply:
personal convenience during implementation. it can be removed later; i don't know the default PS1 on debian, but on alpine it was just "#" or "$", not telling you your directory.